### ISSUE TITLE
Submission message: "All Form Fields" variable does not get printed

### ISSUE THREAD

**Issue by HannahDeWachter** [verbb/formie#1125]
### Describe the bug

Here is what I filled out on the submission message:
<img width="470" alt="Screenshot 2022-10-14 at 13 51 43" src="https://user-images.githubusercontent.com/55497252/195840725-139b69f7-1c9c-4ff5-86fa-16b3b4480bb8.png">

This is the message I see after submitting the form:
<img width="257" alt="Screenshot 2022-10-14 at 13 52 16" src="https://user-images.githubusercontent.com/55497252/195840799-8f8d9d32-1f7d-465b-bf59-63d1ffcb1f4b.png">

### Steps to reproduce

1. Fill in a Submission Message on the form
2. Display form on entry
3. Submit form

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Custom Form Templates: Yes

### Craft CMS version

4.2.5.1

### Plugin version

dev-craft-4 as 2.0.12

### Multi-site?

Yes

**Comment by engram-design** [comment]
What sort of fields does your form contain? Are you overwriting any of the email templates in your custom form templates?

Does the Preview function for the email notification produce anything different?

Lastly, have a look at your `storage/logs/formie.log` file to see if there's any errors logged in there.

**Comment by HannahDeWachter** [comment]
Hi @engram-design ,

The form contains: Name field, multiple "Single-line Text" fields, Phone Number field, multiple Email Address fields, multiple "Multi-line Text" fields and a Checkboxes field, and some Headings and ReCaptcha.

I am not talking about an email, but about the Behaviour of "Action on Submit". The action on submit is "Display a message" with inside the message "All Form Fields" variable which does not get printed.

I don't see any errors in the log file. The only log I see is: `[INFO] Submission triggered for {formHandle}.`

**Comment by engram-design** [comment]
Ah, my mistake - this was the message text after the form is submitted, not for email notifications. Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.12",
  "...": "..."
}
```

Then run `composer update`.

**Comment by HannahDeWachter** [comment]
@engram-design It works! Thank you!

**Comment by engram-design** [comment]
Fixed in [2.0.13](https://github.com/verbb/formie/releases/tag/2.0.13)
